src/lib.rs

```diff
@@ -19,7 +21,7 @@ use log::info;
 
 // Settings
 // NIST recomends 5 rounds for miller rabin. This implementation does 8. Apple uses 16. Three iterations has a probability of 2^80 of failing
-const MILLER_RABIN_ROUNDS: usize = 8usize;
+const MILLER_RABIN_ROUNDS: usize = 64usize;
```

Comment:
I bumped this to 64 to match openssl but we can set it back to 8. Just noticed the comment so I will update it later this evening.